build.sh

```diff
@@ -74,7 +74,9 @@ initTargetDistroRid()
         if [ "$__BuildOS" == "Linux" ]; then
             if [ ! -e $ROOTFS_DIR/etc/os-release ]; then
                 echo "WARNING: Can not determine runtime id for current distro."
-                export __DistroRid=""
+                #Test hack to generate __DistroRid for Android
+                #TODO Find reliable way to generate __DistroRid for Android.
+                export __DistroRid="android.21-arm64"
```

Comment:
IMHO, you need to save `ANDROID_PLATFORM` variable as a file when you build rootfs and then you can use it in building coreclr. :)